orders/src/main/java/com/orders/crudyorders/model/Customer.java

```diff
@@ -0,0 +1,137 @@
+package com.orders.crudyorders.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "customer")
+public class Customer
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(nullable = false)
+    private long custcode;
+
+    @Column(nullable = false)
+    private String custname;
+
+    private String custcity, workingarea, custcountry, grade;
+    private double openingamt, recieveamt, paymentamt, outstandingamt;
+    private String phone;
+
+    @OneToMany
+    @JoinColumn(name = "agentcode", nullable = false)
+    // connects this to the agent model
+    private long agentcode;
+
+    public Customer(){}
+
+    public Customer(String custname, String custcity, String workingarea, String custcountry, String grade, double openingamt, double recieveamt, double paymentamt, double outstandingamt, String phone, long agentcode) {
```

Comment:
Make sure to also include a list of `Order` objects as well.

Ensure that the value of `mappedBy` matches the variable name of the `Customer` object contained in your `Order.java` model class file.

Make sure to ignore the properties of your `Order` object that reference other object types as well to prevent an infinite loop of displaying properties (order displays customer, which displays order, and so forth). This can be done using the `@JsonIgnoreProperties` annotation, which can take in an array of strings (format in Java is `{"string1", "string2"}`).

Example:
```
@OneToMany(mappedBy = "customer",
                cascade = CascadeType.ALL,
                orphanRemoval = true)
@JsonIgnoreProperties({"agent", "customer"})
private List<Order> orders = new ArrayList<>();
```

Lastly, don't forget to include the order list property in your constructor, getters and setters as well.